UTSCCSCC01/finalprojects22-byte-peeps
# Byt 43 you tube data pipeline

- To test:
Run this script to add data to the db: 

```
await User.create({
  username: 'moh',
  password: 'moh',
});

await YouTubeChannel.create({
  channelId: 'UC5uqd7fgT5zoFDGCrUvYdAg',
  name: 'Moh',
  isActive: true,
  userId: 1,
  oauth: '',
});
return;
```

This can be put into a function and run in the `app.ts`. After data has been inserted into the db, the pipeline should run updating the data. Note: will have to invoke the startPipeline function directly as it is pipeline scheduled to run daily.
